src/helm_check_current_release.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 set -eu
+# Ensure Helm release is in a usable state
+# See: https://github.com/kubernetes/helm/issues/4004
```

Comment:
I guess you completely remove these comments

Reply:
Ah you kept the script for archive reasons. Got it ;)

Reply:
No, I'll remove it :ok_hand:  I made this change before I knew it wasn't needed. I first intended to call the script directly in the CI config, that's why it was already removed from the Makefile.